src/main/java/com/spikes2212/util/RepeatCommand.java

```diff
@@ -0,0 +1,34 @@
+package com.spikes2212.util;
+
+import edu.wpi.first.wpilibj2.command.*;
+
+/**
+ * This {@link Command} repeats a sequence of commands indefinitely
+ */
+
+public class RepeatCommand extends CommandBase {
+
+    private Command command;
+
+    public RepeatCommand(Command... command) {
```

Comment:
make `RepeatCommand` require all the subsystems required by its property. 
override `getRequirenments` and make it return the requirenments of `command`